src/Lean/PrettyPrinter/Delaborator/Basic.lean

```diff
@@ -247,6 +247,20 @@ See also `Lean.PrettyPrinter.Delaborator.Context.depth`.
 def withIncDepth (act : DelabM α) : DelabM α := fun ctx =>
   act { ctx with depth := ctx.depth + 1 }
 
+/--
+Returns true if `e` is a "shallow" expression.
+Local variables, constants, and other atomic expressions are always shallow.
+The `threshold` variable adjusts what else is considered to be shallow,
+and it's compared against the depth of the expression.
```

Comment:
"by comparing it against the depth of the expression."? Otherwise, the doc comment suggests that multiple things are happening here.

Reply:
Thanks, these were confusingly worded. I used your ideas and tried refining them even more. How does it look now?

Reply:
I'm happy with your refinement.